util/Migrator/DbScripts/2021-04-16_01_Two_Guid_Array_Type.sql

```diff
@@ -0,0 +1,11 @@
+-- Create TwoGuidIdArray Type
+IF NOT EXISTS (
+    SELECT *
+FROM sys.types
+WHERE [Name] = 'TwoGuidIdArray'
+    AND is_user_defined = 1
+)
+CREATE TYPE [dbo].[TwoGuidIdArray] AS TABLE (
+    [Id1] UNIQUEIDENTIFIER NOT NULL,
+    [Id2] UNIQUEIDENTIFIER NOT NULL);
+GO
```

Comment:
Why did you break the migrator script up into so many parts? Seems more succinct to have one per PR.

Reply:
Honestly, because the script wasn't working as a single statement. I'm not really sure why.

I'm going to also claim in hindsight that it might make reverting something easier, but really it's because it worked better.

Reply:
Could have been the order of execution in the single script, use of `GO` terminators perhaps, etc. Ultimately the first thing should be the new UDT, then stored procedures in order of dependency chain. But this works as well.